<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertExams</class>
 <widget class="QWidget" name="InsertExams">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ispit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_exam_title">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;Podaci o ispitu&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="examLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Odabir predmeta:</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="dateLabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ispit u prvom roku:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox1">
     <property name="text">
      <string>Preskoči unos</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Datum :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dateLineEdit1">
       <property name="placeholderText">
        <string>dd.mm.yyyy.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Vreme :           </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="timeLineEdit1">
       <property name="placeholderText">
        <string>hh:mm</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxOral1">
     <property name="text">
      <string>Podaci o usmenom  ispitu nepoznati (biće dodat u raspored 7 dana nakon pismenog)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Datum usmenog ispita:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dateLineEditOral1">
       <property name="placeholderText">
        <string>dd.mm.yyyy.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Vreme usmenog ispita:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="timeLineEditOral1">
       <property name="placeholderText">
        <string>hh:mm</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ispit u drugom roku:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox2">
     <property name="text">
      <string>Preskoči unos</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Datum :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dateLineEdit2">
       <property name="placeholderText">
        <string>dd.mm.yyyy.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Vreme :           </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="timeLineEdit2">
       <property name="placeholderText">
        <string>hh:mm</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxOral2">
     <property name="text">
      <string>Podaci o usmenom  ispitu nepoznati (biće dodat u raspored 7 dana nakon pismenog)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Datum usmenog ispita:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dateLineEditOral2">
       <property name="placeholderText">
        <string>dd.mm.yyyy.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Vreme usmenog ispita:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="timeLineEditOral2">
       <property name="placeholderText">
        <string>hh:mm</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="errorLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="urlLabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>URL do stranice za proveravanje rezultata:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="urlLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Važnost ispita (1-5 rastuće)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="horizontalSlider">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QPushButton" name="addExamButton">
       <property name="text">
        <string>Dodaj ispit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearWidgetButton">
       <property name="text">
        <string>Očisti prozor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="endInputExamButton">
       <property name="text">
        <string>Završi unos</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
